src/Pos/Explorer/Socket/App.hs

```diff
@@ -123,60 +123,60 @@ notifierServer settings connVar = do
             route [("/socket.io", handler)]
 
 periodicPollChanges
-    :: forall ssc m.
-       (ExplorerMode m, SscHelpersClass ssc)
+    :: forall ssc ctx m.
+       (ExplorerMode ctx m, SscHelpersClass ssc)
```

Comment:
I am looking at it and just want to say that `ssc` parameter doesn't seem to make sense here. You are using `SscGodTossing` anyway,

It's not related to this PR, just saying.